src/components/Books.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Books = () => <h2>Books page</h2>;
+
+export default Books;
```

Comment:
- [ ] Please follows the ducks pattern, `Books.js` should be in a folder called `books`